### ISSUE TITLE
a error from models.py 

### ISSUE THREAD

**Issue by a0923777314** [mdciri/Vox2Vox#5]
in models.py line 55 :
x = encoder_step(x, Nfilter_start*np.power(2,d), ks, False)
shown error as below.
encoder_step() takes from 2 to 3 positional arguments but 4 were given

I don't sure to modify anything.

**Comment by a0923777314** [comment]
thanks for your fix this problem.
but it show different error in the same file.
in models.py line 101:
x = Conv3D(Nf*(2**depth), ks, strides=1, padding='valid', kernel_initializer='he_normal')(x)
shown error as below.
name 'Nf' is not defined

replace Nfilter_start still wrong, please help to fix the code, thank you.

**Comment by mdciri** [comment]
Thanks for the report, I fixed the code.

**Comment by a0923777314** [comment]
Hi, dmciri
there are two problems.

1. in the last issue, I have replaced Nfilter_start to error:" name 'Nf' is not defined"
then it's will show the error below:
Inputs to a layer should be tensors. Got: <tensorflow.python.keras.layers.convolutional.ZeroPadding3D object at 0x000002CC67752488>
please help fix it, thanks.

2. in the paper, it's wrote padding use 'same', but in this code is using valid. It doesn't matter? or I didn't see any detail in the paper?
